packages/ui/src/components/vuestic-sass/icons/_vuestic-icons.scss

```diff
@@ -10,37 +10,37 @@
 .i-nav-notification {
   @extend .i-nav;
 
-  background: url('./../assets/icons/nav-notifications.svg') no-repeat center;
+  background: url('/src/assets/icons/nav-notifications.svg') no-repeat center;
```

Comment:
Do these work for build?

Reply:
Speaking of:
- [ ] docs build
- [ ] ui build
- [ ] vuestic-admin build

Reply:
works for docs & ui